src/main/scala/com/microsoft/partnercatalyst/fortis/spark/dto/FortisItem.scala

```diff
@@ -0,0 +1,7 @@
+package com.microsoft.partnercatalyst.fortis.spark.dto
+
+trait FortisItem {
```

Comment:
Now that we have a clearer idea of the downstream Cassandra schema, we can soon get rid of `originalItem` in favor of more general fields like `title` and `body`. I'll make a follow-up PR once this one is merged.